practice/validator/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Input Validator</title>
+  </head>
+  <body>
+    <header>
+      <h1>Sign-Up Form</h1>
+    </header>
+    <section>
+      <h2></h2>
+      <form id="form-submit">
+        <div>
+          <h3>Name:</h3>
+          <input id="name-input" type="text" name="full-name" placeholder="John Doe">
+          <p>Invalid Entry</p>
+        </div>
```

Comment:
Always use the `<label>` tag to connect text with the description. It helps with accessibility and interaction. Also using an `<h3>` for that role is maybe a slight conflation of style and structure.